streams-bootstrap-core/src/main/java/com/bakdata/kafka/RunningStreams.java

```diff
@@ -49,4 +50,5 @@ public class RunningStreams {
     Topology topology;
     @NonNull
     KafkaStreams streams;
+    HostInfo hostInfo;
```

Comment:
TODO: tests

Reply:
Alternatively we cann add the method to https://github.com/bakdata/streams-bootstrap/blob/9ce7863247d9795b66f06bfc048d5bcfe0122197/streams-bootstrap-core/src/main/java/com/bakdata/kafka/ImprovedStreamsConfig.java and change the field here to ImprovedStreamsConfig

Reply:
hmm... not a bad idea, but I don't like that we would have two fields (`StreamsConfig` and `ImprovedStreamsConfig`) in the same class. This would be a bit confusing. Or do you mean to change the field type of `StreamsConfig` to `ImprovedStreamsConfig`?

Reply:
Change the field time